tests/unit/package_managers/test_gomod.py

```diff
@@ -17,7 +17,6 @@
 from cachi2.core.models.input import Flag, Request
 from cachi2.core.models.output import BuildConfig, EnvironmentVariable, RequestOutput
 from cachi2.core.models.sbom import Component, Property
-from cachi2.core.package_managers import gomod
```

Comment:
Again, as with previous patch, this one feels slightly misplaced, although it may be somewhat related to the parser test module in the next one, so either propose this one as standalone or at least include it at the beginning of the series among other refactors.